lib/compat/wordpress-6.6/class-wp-rest-media-search-handler-gutenberg.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * REST API: WP_REST_Media_Search_Handler class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.6.0
+ */
+
+if ( class_exists( 'WP_REST_Media_Search_Handler' ) ) {
+	return;
+}
```

Comment:
This is now considered a pattern we would like to avoid:

https://github.com/WordPress/gutenberg/blob/trunk/lib/README.md#avoiding-duplicate-declarations

That recommendation is very new and resulted from discussion around the WP 6.5 cycle.

We will need to update to conform to the new recommendation.

Reply:
Thanks for the heads-up, I'll update the PR accordingly.